data/brands/leisure/amusement_arcade.json

```
@@ -36,6 +52,22 @@
         "name": "Warpoint"
       }
     },
+    {
+      "displayName": "湯姆熊歡樂世界",
+      "locationSet": {
+        "include": ["tw"],
+      },
+      "tags": {
+        "brand": "湯姆熊歡樂世界",
+        "brand:en": "Tom's World",
+        "brand:wikidata": "Q15952961",
+        "brand:zh": "湯姆熊歡樂世界",
+        "leisure": "amusement_arcade",
+        "name": "Tom's World",
```

```suggestion
        "name": "湯姆熊歡樂世界",
```

Should the name of Taiwan's one be Chinese?

---

Ok